beancount_n26/__init__.py

```diff
@@ -235,40 +245,62 @@ def extract(self, file_, existing_entries=None):
             for index, line in enumerate(reader):
                 meta = data.new_metadata(file_.name, index)
 
-                s_amount_eur = self._translate('amount_eur')
-                s_amount_foreign_currency = self._translate(
-                    'amount_foreign_currency'
-                )
-                s_payee = self._translate('payee')
-                s_payment_reference = self._translate('payment_reference')
-                s_type_foreign_currency = self._translate(
-                    'type_foreign_currency'
-                )
+                postings = []
 
-                if line[s_amount_eur]:
-                    amount = Decimal(line[s_amount_eur])
-                    currency = 'EUR'
-                else:
-                    amount = Decimal(line[s_amount_foreign_currency])
+                if line[s_amount_foreign_currency]:
+                    exchange_rate = Decimal(line[s_exchange_rate])
+                    amount_eur = Decimal(line[s_amount_eur])
+                    amount_foreign = Decimal(line[s_amount_foreign_currency])
                     currency = line[s_type_foreign_currency]
 
-                match = None
+                    fees = amount_eur + (amount_foreign / exchange_rate)
```

Comment:
I wonder if this formula is coming from the sign difference in the example transaction? I would've assumed it to be something like `fees = amount_eur - (amount_foreign / exchange_rate)`.

Just as another data point, I just checked my CSV exports and one of the transactions there is:

```
"2022-02-05","Mustermann GmbH","","MasterCard Payment","-","-12.21","-12.21","EUR","1.0"
```

... which might break this formula, as the `fees` would amount to `24.21` in that case.

Reply:
Yeah, ignoring the sign of amount_foreign could help to handle that specific case. 

In the case of currency exchanges, I do have the following line :
```
"2022-07-06","[name]","CH46[...]","Virement dans une devise étrangère","[...] ","-2000.0","1970.62","CHF","0.9906"
```


This is fixed in d01c77dc84057a173184d32ff25eeb7e7fc2e715, your example is used in the tests